cmd/genji/shell/command_test.go

```diff
@@ -0,0 +1,82 @@
+package shell
+
+import (
+	"github.com/genjidb/genji"
+	"github.com/stretchr/testify/require"
+	"strings"
+	"testing"
+)
+
+func Test_runTablesCmd(t *testing.T) {
```

Comment:
I think we should have the same way of naming function tests within the codebase. Does it make sense?
```suggestion
func TestTablesCmd(t *testing.T) {
```